internal/command/user_converter.go

```diff
@@ -137,6 +137,9 @@ func writeModelToWebAuthN(wm *HumanWebAuthNWriteModel) *domain.WebAuthNToken {
 }
 
 func authRequestDomainToAuthRequestInfo(authRequest *domain.AuthRequest) *user.AuthRequestInfo {
+	if authRequest == nil {
```

Comment:
Wouldn't an outside if to not even get to this function suffice?

Reply:
i think it's used at 20 places.... so a single check is far more easy